core/cmd/starknet_keys_commands.go

```diff
@@ -13,7 +13,7 @@ type StarkNetKeyPresenter struct {
 
 // RenderTable implements TableRenderer
 func (p StarkNetKeyPresenter) RenderTable(rt RendererTable) error {
-	headers := []string{"ID", "Public key"}
+	headers := []string{"ID", "Account Address", "Stark Key"}
```

Comment:
Why do you think it's important to declare this a "Stark Key". It's just the ECDSA signatures over the STARK-friendly elliptic curve, so we should just label it as a PK here.

```suggestion
	headers := []string{"ID", "Account Address", "Public Key"}
```

Reply:
i'm open to either. I was thinking naming it `Stark Key` and `Account Address` would indicate to the user that this is different the normal `Public Key` flow (like for solana, terra, evm)

also i think a standard ecdsa public key is an X/Y coordinate pair while the stark key is just the X coordinate 🤷🏼 but that's more of a technicality

Reply:
Let's just go w/ `"STARK Public Key"` for now and revisit at the end.